freqtrade/data/history/hdf5datahandler.py

```diff
@@ -157,7 +189,11 @@ def trades_append(self, pair: str, data: TradeList):
         """
         raise NotImplementedError()
 
-    def _trades_load(self, pair: str, timerange: Optional[TimeRange] = None) -> TradeList:
+    def _trades_load(
+        self,
+        pair: str,
+        timerange: Optional[TimeRange] = None,
+    ) -> TradeList:
```

Comment:
Please don't make pointless formatting changes.

I think i mention this in every other PR - please keep this in mind - i'm tired of repeating my self over and over and over again with the same thing.